plugin/core/main.py

```diff
@@ -1,3 +1,5 @@
+from LSP.plugin.highlights import remove_highlights
+from LSP.plugin.color import remove_color_boxes
```

Comment:
Have you tried ` from ..color import remove_color_boxes` instead?

Reply:
I haven't. 
The whole LSP project uses relative imports, 
so I guess I should be consistent.